File: start_arg/sample0_05.java
public class sample0_05{
		public static void main(String[] args){
			int [] array={12,13,11,14,10};
			int max=array[0];
			for(int i=1;i<array.length-1;i++){
				if(array[i]>max){
					max=array[i];
				}
			}
			System.out.println(max);
		}
}

File: start_arg/sample0_05.php
<?php
/*
最大値を探しだすアルゴリズム
5つのデータのうち一番大きいのはどれか
*/
$array=array(12,13,11,14,10);
$max=$array[0];
for($i=1;$i<count($array)-1;$i++){
	
	if($array[$i]>$max){
		$max=$array[$i];
	}
}
echo $max;
?>